### ISSUE TITLE
rubygem faraday_middleware-0.12.1 broken

### ISSUE THREAD

**Issue by jayvdb** [coala/docker-coala-base#234]
```
The latest bundler is 1.15.3, but you are currently running 1.15.0.

To update, run `gem install bundler`

Bundler cannot continue installing faraday_middleware (0.12.1).

The checksum for the downloaded `faraday_middleware-0.12.1.gem` does not match
the checksum given by the server. This means the contents of the downloaded gem
is different from what was uploaded to the server, and could be a potential
security issue.

To resolve this issue:

1. delete the downloaded gem located at:
`/usr/lib64/ruby/gems/2.2.0/gems/faraday_middleware-0.12.1/faraday_middleware-0.12.1.gem`
2. run `bundle install`

If you wish to continue installing the downloaded gem, and are certain it does
not pose a security issue despite the mismatching checksum, do the following:

1. run `bundle config disable_checksum_validation true` to turn off checksum
verification
2. run `bundle install`

(More info: The expected SHA256 checksum was
"06764c0b5e7f10a5efeae501da2c825f0e479969c8ca9b585583fd6546ba63df", but the
checksum for the downloaded gem was
"021dc9fd7e7b6de1bc394637c7b8214a1f8b2d1407867e5239c90b5405a31ff3".)
```

https://travis-ci.org/jayvdb/docker-coala-base/builds/260568358#L2508

**Comment by jayvdb** [comment]
Problem here : https://github.com/lostisland/faraday_middleware/issues/165

**Comment by jayvdb** [comment]
Fixed upstream with 0.12.2
